sinks/kafka.go

```diff
@@ -60,19 +61,28 @@ func (k *Kafka) Start(_ ...interface{}) error {
 	resendChan := make(chan *sarama.ProducerMessage, 10000)
 	// Notice order could get altered having more than 1 producer
 	log.WithFields(log.Fields{
+		"Name":       k.name,
 		"NbProducer": k.kafkaConf.NbProducer,
 	}).Debug("Starting sink producers")
 	for x := 0; x < k.kafkaConf.NbProducer; x++ {
-		go startProducer(k.kafkaConf, resendChan, k.stop)
+		go k.startProducer(resendChan, k.stop)
 	}
 
 	//current kafka threshold is 10MB
-	threshold := 10 << 10
+
 	log.WithFields(log.Fields{
-		"threshold": threshold,
+		"threshold": k.kafkaConf.MaxMessageBytes,
 	}).Debug("KafkaSink: started with threshold")
 
-	go startConsumer(k.kafkaConf, k.in, threshold, resendChan)
+	go k.startConsumer(resendChan)
+
+	//Send empty event every Minutes as to flush buffer
+	ticker := time.NewTicker(time.Minute * 1)
```

Comment:
* 1 is not usefull. If it's destined to be modified in the future, it should be put into constants